docs-src/references/configuration.md

```diff
@@ -387,3 +387,123 @@ publicClient:
+- `enableRead`: Supported values are `true` or `false`. Set to `true` to allow read operations from the archived Event History data.
+- `provider`: Location where data should be archived. Subprovider configs are `filestore`, `gstorage`, `s3`, or `your_custom_provider`. Default configuration specifies `filestore`.
+
+Example:
+
+```yaml
+# Cluster level Archival config
+archival:
+  # Event History configuration
+  history:
+    # Archival is enabled for the History service data
+    state: "enabled"
+    enableRead: true
+    # Namespaces can use either the local filestore provider or the Google Cloud provider
+    provider:
+      filestore:
+        fileMode: "0666"
+        dirMode: "0766"
+      gstorage:
+        credentialsPath: "/tmp/gcloud/keyfile.json"
+  # Configuration for archiving Visibility data.
+  visibility:
+    # Archival is enabled for Visibility data
+    state: "enabled"
+    enableRead: true
+    provider:
+      filestore:
+        fileMode: "0666"
+        dirMode: "0766":
+```
+
+For more details on Archival setup, see [Set up Archival](/cluster-deployment-guide#set-up-archival).
+
+## namespaceDefaults
+
+_Optional_
+
+Sets default Archival configuration for each Namespace using `namespaceDefaults` for `history` and `visibility` data.
+
+- `state`: Default state of the Archival for the Namespace. Supported values are `enabled` or `disabled`.
+- `URI`: Default URI for the Namespace.
+
+For more details on setting Namespace defaults on Archival, see [Namespace creation in Archival setup](/cluster-deployment-guide#namespace-creation)
+
+Example:
+
+```yaml
+# Default values for a Namespace if none are provided at creation
+namespaceDefaults:
+  # Archival defaults
+  archival:
+    # Event History defaults
+    history:
+      state: "enabled"
+      # New Namespaces will default to the local provider
+      URI: "file:///tmp/temporal_archival/development"
+    visibility:
+      state: "disabled"
+      URI: "file:///tmp/temporal_vis_archival/development"
+```
+
+## dcRedirectionPolicy
+
+_Optional_
+
+Contains the Frontend datacenter API redirection policy that you can use for cross-DC replication.
+
+Supported values are:
+
+- `policy`: Supported values are `noop`, `selected-apis-forwarding`, `all-apis-forwarding`.
+  - `noop`: Not setting a value or setting `noop` means no redirection. This is the default value.
+  - `selected-apis-forwarding`: Sets up forwarding for the following APIs to the active Cluster based on the Namespace:
+    - `StartWorkflowExecution`
+    - `SignalWithStartWorkflowExecution`
+    - `SignalWorkflowExecution`
+    - `RequestCancelWorkflowExecution`
+    - `TerminateWorkflowExecution`
+    - `QueryWorkflow`
+      See [SDK documentation](/dev-guide) for details.
+  - `all-apis-forwarding`: Sets up forwarding for all APIs on the Namespace in the active Cluster.
+
+Example":
+
+```yaml
+#...
+dcRedirectionPolicy:
+  policy: "selected-apis-forwarding"
+  toDC: ""
```

Comment:
We are removing this in next release: https://github.com/temporalio/temporal/pull/4370

Reply:
yes; have removed the field.